test/support/distribution_support.ex

```diff
@@ -0,0 +1,57 @@
+defmodule EctoPSQLExtras.DistributionSupport do
+  # This module helps to spawn a new node to test
+  # distribution. The remote node does not requires
+  # EctoPSQLExtras, but requires that the repository
+  # exists.
+  #
+  # Check the "test/support/dummy_app.exs".
+  # It requires Elixir 1.12
+  def setup_support_project!(filename) do
```

Comment:
Erlang actually has a functionality to build this. It is in the very poorly named module called `slave`. Here is how Phoenix PubSub uses it to spawn new nodes: https://github.com/phoenixframework/phoenix_pubsub/blob/master/test/support/cluster.ex#L1

Reply:
@josevalim Cool! And yeah, poorly named indeed :sweat: 

But I think is not quite the same I want here. My idea is to start a node without the Ecto PSQL Extra code, but with Ecto and a simple repository.
Do you think this is too much? I mean: should I only focus on having a remote node with the same code instead?

Reply:
You can control exactly what is loaded on the other node. Just make sure you remove ecto_psql stuff from the code path here: https://github.com/phoenixframework/phoenix_pubsub/blob/master/test/support/cluster.ex#L18

Reply:
Oh, I see! I will try that later today. Thanks! :D

Reply:
@josevalim Done! Sorry for the delay